metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -379,6 +383,14 @@ def build_corp_user_mce(
             if self.config.user_attrs_map["title"] in attrs
             else None
         )
+        custom_props_map = {}
+        if self.config.custom_props_list:
+            for prop in self.config.custom_props_list:
+                if self.config.user_attrs_map.get(prop) in attrs:
+                    custom_props_map[prop] = (
+                        attrs[self.config.user_attrs_map[prop]][0]
```

Comment:
This would require each item in `custom_props_list` to also be included in `user_attrs_map`. Additionally, user_attrs_map is generally a mapping of datahub name -> custom attr name, not the reverse. I think we should just use the items in custom_props_list directly

Reply:
>This would require each item in custom_props_list to also be included in user_attrs_map. Additionally, >user_attrs_map is generally a mapping of datahub name -> custom attr name, not the reverse. I think we should just >use the items in custom_props_list directly.

I use "user_attrs_map"  to map datahub property to an actual ldap property. For example, I have this mapping in my "user_attrs_map": {"countryCode":"c"}. I do this to replace a default datahub mapping for "countryCode", which is "countryCode":"countryCode", by an actual ldap attribute for my company, which is "c".

custom_props_list is just a list of my custom datahub properties for corpUser, not an actual ldap attributes name. For example, in my case I want to use "accountType" as custom name property. This is why I use "user_attrs_map", I need to map "accountType" to an actual ldap attribute. 

I think instead you suggest to use ldap attribute name directly at custom_props_list and not to use aliases. This could be done, but I believe we are going to have some logic in UI build on values of  some custom properties, not just display them. In this case, if ldap attribute name is changed, we will need to rename all place in UI code with a hardcoded previous ldap attribute name and we want to avoid this by using aliases.

Reply:
@hsheth2, I am going to make an update you have suggested.